composer/algorithms/gyro_dropout/README.md

```diff
@@ -0,0 +1,79 @@
+# 🍰 Gyro Dropout
+
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+ `Computer Vision`
+
+Gyro Dropout replaces implementations of `torch.nn.Dropout`. The Gyro Dropout provides increased accuracy compared with dropout.
+
+Gyro dropout is a variant of dropout that improves the efficiency of training neural networks
+Instead of randomly dropping out neurons in every training iteration, gyro dropout pre-selects and trains a fixed
```

Comment:
```suggestion
Gyro dropout is a variant of dropout that improves the efficiency of training neural networks.
Instead of randomly dropping out neurons in every training iteration, gyro dropout pre-selects and trains a fixed
```